torch/distributed/_tensor/op_schema.py

```diff
@@ -268,9 +276,10 @@ def arg_type_tensor_or_tensor_list_like(self, arg_idx: int) -> bool:
 
         return all(isinstance(e, DTensorSpec) or e is None for e in arg)
 
-    def return_type_tuple_tensors(self) -> bool:
+    def return_type_tuple_tensor_like(self) -> bool:
+        # all dispatch ops could only return Tuple[Tensor] or have None/ints/floats
+        # in the tuple, but the first element must be a Tensor, so this check is enough
```

Comment:
Also for my learning, why the first element must be a tensor?

Reply:
This is the current pattern of all aten ops, if it returns a tuple, the first element is torch.tensor for all cases